tools/ci_connector_ops/ci_connector_ops/pipelines/contexts.py

```diff
@@ -67,6 +67,8 @@ def __init__(
         slack_webhook: Optional[str] = None,
         reporting_slack_channel: Optional[str] = None,
         pull_request: PullRequest = None,
+        ci_report_bucket: Optional[str] = None,
+        ci_gcs_credentials: Optional[str] = None,
```

Comment:
I moved pipeline report logic and creds to the base pipeline

Reply:
nit: Do we want to persist reports for metadata pipelines?
I believe dagger metadata pipelines  running in the CI will soon be removed as metadata validation will happen in test.